src/js/components/GithubUser/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import { Loader } from 'semantic-ui-react';
+import { Img, Btn } from './style';
+
+class GithubUser extends PureComponent {
+  getProfile = () => {
+    const { fetchGithubUser } = this.props;
+    fetchGithubUser();
+  };
+
+  render() {
+    const { profile, err, loading } = this.props;
+
+    if (!isEmpty(err)) {
+      return (
+        <div>
+          {err.response.status}
+          {err.response.statusText}
+        </div>
+      );
+    }
+
+    if (loading === true) {
+      return <Loader active inline />;
+    }
+
+    if (loading === false && !isEmpty(profile)) {
+      return (
+        <div>
+          <Img src={profile.avatar_url} alt="github icon" />
+          <Btn type="submit" onClick={this.getProfile}>
+            get profile
+          </Btn>
+        </div>
+      );
+    }
+    return (
+      <div>
+        <Btn type="submit" onClick={this.getProfile}>
+          get profile
+        </Btn>
+      </div>
+    );
+  }
+}
```

Comment:
'render' method looks kind of messy because of many `if` conditions, is there any way to make this more clear code?

Reply:
Yes, have other way to do this:
`
render() {
    const { profile, err, loading } = this.props;

     if (!isEmpty(err)) {
      return (
        <div>
          {err.response.status}
          {err.response.statusText}
        </div>
      );
    }

    return (
      {
        loading
          ? (<Loader active inline />)
          : (<div>
              {!isEmpty(profile) && (<Img src={profile.avatar_url} alt="github icon" />)}
              <Btn type="submit" onClick={this.getProfile}>
                get profile
              </Btn>
            </div>)
      }
    );
  }
}

`

Reply:
Yes have other way to do this:
<img width="848" alt="screen shot 2018-09-18 at 10 05 22" src="https://user-images.githubusercontent.com/15276422/45662118-c0a72480-bb2a-11e8-8dd4-900741fb1687.png">